diff --git a/lib/mix/tasks/esbuild.ex b/lib/mix/tasks/esbuild.ex
index efee41e..56da05b 100644
--- a/lib/mix/tasks/esbuild.ex
+++ b/lib/mix/tasks/esbuild.ex
@@ -44,7 +44,7 @@ defmodule Mix.Tasks.Esbuild do
     if opts[:runtime_config] do
       Mix.Task.run("app.config")
     else
-      Mix.Task.run("loadpaths")
+      Mix.Task.run("loadpaths", ["--no-deps-check", "--no-elixir-version-check"])
       Application.ensure_all_started(:esbuild)
     end
 
